scripts/check-links.js

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const https = require('https');
+const http = require('http');
+const { URL } = require('url');
+const { execSync } = require('child_process');
+
+// ANSI color codes for output
+const colors = {
+  red: '\x1b[31m',
+  green: '\x1b[32m',
+  yellow: '\x1b[33m',
+  reset: '\x1b[0m'
+};
+
+// Find all README files
+function findReadmeFiles(excludeTestFixtures = true) {
+  try {
+    let command = 'find . -name "README.md" -type f | grep -v node_modules';
```

Comment:
Can't you just read the `packages` folder and check if there is a README.md file in each result?